cmd/scaffold.go

```diff
@@ -135,6 +136,7 @@ type FlowNodeBuilder struct {
 	postInitFns       []func(*FlowNodeBuilder)
 	stakingKey        crypto.PrivateKey
 	networkKey        crypto.PrivateKey
+	Unstaked          bool
```

Comment:
This is the flag used to determine if the node should run as staked (`Unstaked=false`) or unstaked (`Unstaked=true`). Although it is only used for an access node, it is define here in scaffold because it needs to influence how the node is initialized by the flow node builder.